stub-environment/roles/Hannibal-Deploy.json

```diff
@@ -4,8 +4,12 @@
     },
     "json_class": "Chef::Role",
     "run_list": [
-      "recipe[bcpc-hadoop::hannibal]",
-      "recipe[hannibal::hannibal_deploy]"
+      "recipe[bach_hannibal::default]",
+      "recipe[bach_hannibal::hannibaldb]",
+      "recipe[bach_hannibal::hannibalhb]",
+      "recipe[hannibal::default]",
```

Comment:
After discussing with Amit:

Let's move all the `hannibal::*` recipes inside the role to `include_recipe 'hannibal::*'` calls inside the `bach_hannibal` wrapper cookbook.

It's better to have the core recipes in `hannibal` to be explicitly depended on to have less surprises.